dts/bindings/flash_controller/flash-pages-layout.yaml

```diff
@@ -0,0 +1,15 @@
+include: base.yaml
+
+compatible: "flash-pages-layout"
```

Comment:
A generic binding like this being added for a specific SoC without any justification in the commit message? No way. Either justify this or do something specific in your own binding, like a `child-binding` or additional array properties.

Reply:
thanks, I updated to use child-binding